Misc/NEWS.d/next/Library/2022-02-08-16-42-20.bpo-46066.m32Hl0.rst

```diff
@@ -0,0 +1 @@
+Remove kwargs-based syntax for TypedDict definition
```

Comment:
You should feel free to add your name to the news entry, if you like ("Patch by XXX"), and also add your name to https://github.com/python/cpython/blob/main/Misc/ACKS (both are optional, only if you want to 🙂)

```suggestion
Deprecate kwargs-based syntax for :class:`typing.TypedDict` definitions.
It was unsupported by type checkers.
```